types/streamify-string/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for streamify-string 1.0
+// Project: https://github.com/chrisallenlane/streamify-string
+// Definitions by: Jesse Wright <https://github.com/jeswr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Readable } from 'stream';
+
+interface Streamify extends Readable {}
+interface StreamifyConstructor {
+  new (str: string, options?: {}): Streamify;
+  (str: string, options?: {}): Streamify;
```

Comment:
`{}` was just a pseudocode, you may want to update this to `any` or whatever are the options in the package